modules/vba_documents/app/swagger/vba_documents/v1/description.md

```diff
@@ -56,5 +56,11 @@ A request to the `/uploads/{id}` endpoint will return a real-time status for tha
 
 The `updated_at` field indicates the last time the status for a given GUID was updated.
 
+### Optional Base64 Encoding
+
+Base 64 is an encoding scheme that converts binary data into text format so that encoded textual data can be easily transported over network un-corrupted and without any data loss. Base64 is used to take our binary multipart/form-data and encode it in it's entirety in order to bypass certain false positives encountered when going through the VA Firewall. If you feel like you are getting a greater number of PDF files being rejected when uploading them through the PUT request then you might consider encoding the entire payload in base64.
```

Comment:
```suggestion
Base64 is an encoding scheme that converts binary data into text format, so that encoded textual data can be easily transported over network uncorrupted and without data loss. 

Base64 can be used to encode binary multipart/form-data it in its entirety, in order to bypass certain false positives encountered as the VA firewall scans for attack signatures. If your PDF file uploads are failing, causing your GUIDs to expire before a PDF is received, you might consider encoding the entire payload in base64.
```